- Don't unconditionally strip the binary.
- Add missing CFLAG.
- Recognize IABSD as BSD.

--- configure-alt.orig	Tue Apr  8 10:54:24 2014
+++ configure-alt	Thu Apr 10 13:48:26 2014
@@ -66,6 +66,7 @@ case `uname` in
 	FreeBSD) true ${os_type:=_BSD} ;;
 	NetBSD) true ${os_type:=_BSD} ;;
 	OpenBSD) true ${os_type:=_BSD} ;;
+	IABSD) true ${os_type:=_BSD} ;;
 	SunOS) true ${os_type:=SUNOS} ;;
 	AIX) true ${os_type:=AIX} ;;
 	OSF1) true ${os_type:=TRU64} ;;
@@ -474,7 +475,7 @@ fi
 cat > makefile_build <<EOF
 # makefile for newLISP v. 10.1.x generated by the configure script (configure-alt)
 CC = ${CC:-${DEFAULT_CC}}
-CFLAGS = ${CFLAGS:-${DEFAULT_CFLAGS}} ${INCLUDES}
+CFLAGS = ${CFLAGS:-${DEFAULT_CFLAGS}} -c ${INCLUDES}
 LDFLAGS = ${LDFLAGS:-${DEFAULT_LDFLAGS}}
 OBJS = ${OBJS:-${DEFAULT_OBJS}}
 STRIP = ${STRIP:-${DEFAULT_STRIP}}
@@ -485,7 +486,6 @@ ARCHFLAGS = ${ARCHFLAGS:-${DEFAULT_ARCHFLAGS}}

 default: \$(OBJS)
 ${LINKSTAGE}
-${STRIPBINARY}

 .c.o:
 ${COMPILEOBJS}
